tests_e2e/pipeline/scripts/execute_tests.sh

```diff
@@ -36,7 +36,7 @@ sudo chown "$WAAGENT_UID" "$LOGS_DIRECTORY"
 #
 az acr login --name waagenttests --username "$CR_USER" --password "$CR_SECRET"
 
-docker pull waagenttests.azurecr.io/waagenttests:latest
+docker pull waagenttests.azurecr.io/waagenttests-mariner:version1.0
```

Comment:
pushed new image under this name

Reply:
we should keep the same name, we don't need the previous image

Reply:
sure. But I would maintain history. So, I would keep it as waagenttests:version2.0

Reply:
ok, if keeping a history please find a way to associate images in the repository with their corresponding Dockerfile

Reply:
There is an option to retrieve the dockerfile from an image. https://hub.docker.com/r/alpine/dfimage. This will do it, no need to link

Reply:
can you run the tool and compare the generated file with the original -- comments will be lost for sure

Reply:
yes, you are right, no comments included and also missing FROM statement too. I don't find any other way we could tag dockerfile

Reply:
i guess you could just add a comment in the Dockerfile specifying the image name, let's say

#
# NOTE: Update this comment when publishing a new image
#
# Published as  "waagenttests.azurecr.io/waagenttests:1.0"
#

Reply:
since you are planning on keeping a history, i'm just trying to avoid a situation where 6 months from now we have multiple images in the repository and we have no idea what they are.

Reply:
My idea is saving images from unintended push. If we maintain version tags someone need to know the versions to update it.

Reply:
I agree we don't have a use case where multiple consumers need different versions something like that. As we are only the consumer so there is not lot of use with it. I'm ok to keep as before. My original intention was that.

Reply:
docker keeps a history already

Reply:
I'll keep as it before waagenttests.azurecr.io/waagenttests:latest. If my changes look good, I'll push the image.

Reply:
you tested the image before submitting the PR, right?

Reply:
yup, I did. Pipeline run https://dev.azure.com/cplatruntime/WALinuxAgent/_build/results?buildId=7387&view=logs&j=fe5f4804-78d2-575e-feb3-7b106cfc303e&t=9d896e20-0fff-545e-9e48-170cc501fac9

Reply:
ok, feel free to push it